tests/test_monitor_helpers.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+import pandas as pd
+
+from datetime import datetime
+from cosmo.monitor_helpers import convert_day_of_year, fit_line, explode_df, AbsoluteTime
+
+
+class TestConvertDayofYear:
+
+    def test_mjd(self):
+        test_date = 2019.100
+        test_mjd = 58484.0
+
+        result = convert_day_of_year(test_date, mjd=True)
+
+        assert isinstance(result, float) is True
+        assert result == test_mjd
+
+    def test_datetime(self):
+        test_date = 2017.301
+        test_datetime = datetime(2017, 10, 28, 0, 0)
+
+        result = convert_day_of_year(test_date)
+
+        assert isinstance(result, datetime)
+        assert result == test_datetime
+
+    def test_date_type_fail(self):
+        with pytest.raises(TypeError):
+            convert_day_of_year(1)  # date can only be a string or float. This should fail
```

Comment:
Would be better to pass the type to be tested as a param to ``test_date_type_fail`` and parameterize.